assembl/static/js/app/views/messageColumnsPanel.js

```diff
@@ -395,6 +403,11 @@ var MessageColumnView = BaseMessageColumnView.extend({
       }else{
         $('.js_messageColumnDescription').removeClass('message-column-description');
       }
+      if(!synthesis_title || synthesis_title.isEmptyStripped(translationData)){
+        $('.js_messageColumnSynthesisTitle').addClass('message-column-description');
+      }else{
+        $('.js_messageColumnSynthesisTitle').removeClass('message-column-description');
+      }
```

Comment:
I'm not sure I understand the logic/goal behind this block.

Reply:
It's just for style, it adds or removes a border and margin.